src/delta.rs

```diff
@@ -314,22 +322,40 @@ fn handle_hunk_line(
     {
         painter.paint_buffered_lines();
     }
+    let mut line_numbers = Vec::new();
     match line.chars().next() {
         Some('-') => {
             if state == State::HunkPlus {
                 painter.paint_buffered_lines();
             }
             painter.minus_lines.push(prepare(&line, true, config));
+            line_numbers.push(prepare_line_numbers(
+                Some(painter.line_number_in),
+                None,
+                config,
+            ));
             State::HunkMinus
         }
         Some('+') => {
             painter.plus_lines.push(prepare(&line, true, config));
+            line_numbers.push(prepare_line_numbers(
+                None,
+                Some(painter.line_number_out),
+                config,
+            ));
```

Comment:
Are these ones (in the `-` and `+` branches) necessary?

Reply:
Nope! Removed.